src/wasm/flb_wasm.c

```diff
@@ -70,8 +70,11 @@ static int flb_wasm_load_wasm_binary(const char *wasm_path, int8_t **out_buf, ui
     return buffer != NULL;
 
 error:
+    if (buffer != NULL) {
+        BH_FREE(buffer);
+    }
```

Comment:
This is to fix following error.
It occurs when invalid `Wasm_Path` is set.
e.g. `Wasm_Path fluent-bit/examples/filter_wasm_go/filter.go`

```
==12705== 812 bytes in 4 blocks are definitely lost in loss record 1 of 1
==12705==    at 0x4848899: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==12705==    by 0xC4418E: flb_malloc (flb_mem.h:79)
==12705==    by 0xC48B42: wasm_runtime_malloc_internal (wasm_memory.c:104)
==12705==    by 0xC48CBC: wasm_runtime_malloc (wasm_memory.c:156)
==12705==    by 0xC48801: bh_read_file_to_buffer (bh_read_file.c:96)
==12705==    by 0xC47BC5: flb_wasm_load_wasm_binary (flb_wasm.c:48)
==12705==    by 0xC47F42: flb_wasm_instantiate (flb_wasm.c:132)
==12705==    by 0x6D95A8: cb_wasm_filter (filter_wasm.c:70)
==12705==    by 0x1FE970: flb_filter_do (flb_filter.c:138)
==12705==    by 0x2C82A3: input_chunk_append_raw (flb_input_chunk.c:1624)
==12705==    by 0x2C8A1F: flb_input_chunk_append_raw (flb_input_chunk.c:1881)
==12705==    by 0x2CC7AB: input_log_append (flb_input_log.c:33)
```